Makefile

```diff
@@ -112,9 +113,24 @@ api-docs: gen-crd-api-docs
 	docker run --rm --volume "`pwd`:/data:Z" pandoc/minimal -f html -t rst --lua-filter=/data/hack/ref_links.lua \
 	--columns 200 /data/build/_output/crds-api.html -o /data/docs/customizations/crds.rst
 
+.PHONY: build-cache
+build-cache:
+	@if [ -d "$(CACHE_DIR)" ]; then \
+		echo "$(CACHE_DIR) exists. Skipping build-cache."; \
```

Comment:
can you add a clean target which will delete this cache dir ?

Reply:
Done